index.bs

```diff
@@ -11289,45 +11322,57 @@ being thrown.  When not in strict mode, the assignment attempt will be ignored.
-    the interface, and the [=relevant Realm=] of the object that is the location of the property.
+For each unique [=identifier=] of an [=exposed=] [=operation=] defined on the [=interface=],
+there must exist a corresponding property,
+unless the [=operation=] has [=no allowable invocation=].
 
-    Note: that is, even if an [=includes statement=] was used
-    to make an operation available on the interface,
-    we pass in the [=interface=] which includes the [=interface mixin=],
-    and not the [=interface mixin=] on which the operation was originally declared.
+[=Static operations=] are exposed of the [=interface object=].
+[=Regular operations=] are exposed on the [=interface prototype object=],
+unless the operation is [=unforgeable=] or
+the interface was declared with the [{{Global}}] [=extended attribute=],
+in which case they are exposed on every object that implements the interface.
 
-<p class="advisement">
-    The above description has some bugs, especially around partial interfaces. See
-    <a href="https://github.com/heycam/webidl/issues/164">issue #164</a>.
-</p>
+<div algorithm>
+    To <dfn>expose the regular operations</dfn> of [=interface=] |interface| on |target|, given [=Realm=] |realm|, 
+    run the following steps:
+
+    1.  Let |operations| be the [=list=] of [=exposed=] [=regular operations=] that are [=members=] of |interface|.
+    1.  [=list/Remove=] from |operations| all the [=operations=] that are [=unforgeable=].
+    1.  [=Expose the operations=] |operations| of |interface| on |target| given |realm|.
+</div>
 
-For [=namespaces=], the properties corresponding to each declared operation are described in
-[[#namespace-object]]. (We hope to eventually move interfaces to the same explicit
-property-installation style as namespaces.)
+<div algorithm>
+    To <dfn>expose the static operations</dfn> of [=interface=] |interface| on |target|, given [=Realm=] |realm|, 
+    run the following steps:
+
+    1.  Let |attributes| be the [=list=] of [=exposed=] [=static operations=] that are [=members=] of |interface|.
+    1.  [=Expose the operations=] |operations| of |interface| on |target| given |realm|.
+</div>
+
+<div algorithm>
+    To <dfn>expose the unforgeable regular operations</dfn> of [=interface=] |interface| on |target|, given [=Realm=] |realm|, 
+    run the following steps:
+
+    1.  Let |operations| be the [=list=] of [=exposed=] [=unforgeable=] [=regular operations=] that are [=members=] of |interface|.
+    1.  [=Expose the operations=] |attributes| of |interface| on |target| given |realm|.
+</div>
+
+<div algorithm>
+    To <dfn>expose the operations</dfn> |attributes| of [=interface=] |interface| on |target|
+    given [=Realm=] |realm| run the following steps:
+
+    1.  [=list/For each=] [=operation=] |op| of |operations|:
+        1.  If |op| has [=no allowable invocations=], then [=iteration/continue=].
```

Comment:
I don't know. I'm keeping the original normative statements around here because I don't know better and want to keep the PR sort of manageable. That said, if this is something we can safely drop, I'm happy to do so.

Reply:
Removing as per #511.